typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -476,12 +476,20 @@ export default class Axes2DLayer extends Layer<Axes2DLayerProps> {
 
         const zDepth = zDepthAxesAxis;
 
+        const isB = this.props.isBottomRuler;
+        const isT = this.props.isTopRuler;
+        const isL = this.props.isLeftRuler;
+        const isR = this.props.isRightRuler;
+
         //- BOTTOM RULER ----------------------------------------
-        if (this.props.isBottomRuler) {
-            const axes = [xMin, yMin + mv, zDepth, xMax, yMin + mv, zDepth];
+        if (isB) {
+            const xmin = xMin + (isL ? mh : 0);
```

Comment:
`xmin`, `xmax`, etc. calculation is duplicated, and it's confusing to have variable pairs `xmin`, `xMin`, etc. named so similarly.